src/main/java/com/gibraltar/iberia/challenge/StoneChallenge.java

```diff
@@ -63,13 +57,14 @@ public void preInit(FMLPreInitializationEvent event) {
 	public void loadConfig(Configuration config) {
 		super.loadConfig(config);
 
+		//what even is going on here?
```

Comment:
This defaults the tool material types for slowdown, if they haven't been set yet. Then below (line 67+), we read them all.

Reply:
Ah. Figured there must have been a reason. Just seems weird assigning them all to the same variable.